File: test/fixtures/transforms/model-swap-simple/expected.py
import anthropic

client = anthropic.Anthropic()


def summarize(text: str) -> str:
    message = client.messages.create(
        model="claude-sonnet-4-6",
        max_tokens=1024,
        messages=[{"role": "user", "content": "Summarize this:\n\n" + text}],
    )
    return message.content[0].text

File: test/fixtures/transforms/js-boundary-no-swap/expected.js
import OpenAI from "openai";

const openai = new OpenAI();

export async function draft(question) {
  const completion = await openai.chat.completions.create({
    model: "gpt-4-turbo-preview",
    messages: [{ role: "user", content: question }],
  });
  return completion.choices[0].message.content ?? "";
}
